index.html

```diff
@@ -1,10 +1,192 @@
+                        <h3>User Experience</h3></div>
+                    <p>In order to build successful products you have to understand the user. We create user personas, wireframes and prototypes to test with users. This allows us to gather feedback which helps to shape the product
+                        and
+                        can
+                        quickly
+                        eliminate potential risks – saving you time and money.</p>
+                </article>
+
+                <article class="serviceblock sbspecial" id="logo-design">
+                    <div class="img-for-card"><img src="assets/icon_graphic-design.svg">
+                        <h3>Logo Design + Branding</h3></div>
+                    <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and brand identity. The best brands stand out from the crowd by having a clear and unique point of
+                        view.
+                        We
+                        will work
+                        with you to understand your vision and help create a unique brand identity and style guide for your company.</p>
+                </article>
+
+                <article class="serviceblock sbspecial" id="seo">
+                    <div class="img-for-card"><img src="assets/icon_seo.svg">
+                        <h3>SEO</h3></div>
+                    <p>Do you want your site to rank higher in search results? Google is constantly changing their algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make sure your website it optimized,
+                        providing
+                        you
+                        the best
+                        opportunities for organic search exposure.</p>
+                </article>
+
+                <article class="serviceblock sbspecial" id="copywriting">
+                    <div class="img-for-card"><img src="assets/icon_copywriting.svg">
+                        <h3>Copy Writing</h3></div>
+
+                    <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique tone of voice will help you stand out from the crowd and build brand loyalty. We work with you to establish your brand
+                        bible
+                        and
+                        ensure your
+                        messaging is focused and targeted.</p>
+                </article>
+
+
+            </section>
+
+            <section class="formarea">
+                <div class="form-title-wrapper">
+
+                    <h1>Let's work together</h1>
+
+                    <p class="contact-link" id="contact-white">Have a question? Need a quote? Let us know!</p>
+                </div>
+
+                <form action="#" method="POST" class="contactform">
+                    <!--                    <fieldset class="personalia">-->
+                    <ul class="form-outer-wrapper">
```

Comment:
Labels en inputs in een lijst? 🤔  Erg origineel, maar niet de bedoeling 🤣 Je kunt labels een `display: block` geven als je graag wil dat ze onder elkaar komen te staan ;)